Sprint-2/debug/0.js

```diff
@@ -1,7 +1,16 @@
 // Predict and explain first...
 
+// function multiply(a, b) {
+//   console.log(a * b);
+// }
+
+// console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
```

Comment:
It would be very beneficial for your learning to formally write out your prediction before changing the code.